README.md

```diff
@@ -31,7 +31,7 @@ and much more.
 
 ## Development
 
-You can start a Velum development environment by following the instructions in [caasp-kvm](https://github.com/kubic-project/automation/caasp-kvm).
+You can start a Velum development environment by following the instructions in [caasp-kvm](https://github.com/kubic-project/automation).
```

Comment:
Shouldn't it be `https://github.com/kubic-project/automation/tree/master/caasp-kvm`?

Reply:
@vitoravelino in theory yes, but i didn't put it the full link, because maybe we will change the `directory` or the location or something, and we have again a broken link

Reply:
yep true, changing text